tests/modules/programs/lsd/example-settings.nix

```
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.lsd = {
+      enable = true;
+      enableAliases = false;
+      settings = {
+        date = "relative";
+        blocks = [ "date" "size" "name" ];
+        color.theme = "custom";
+        layout = "oneline";
+        sorting.dir-grouping = "first";
+        ignore-globs = [
+          ".git"
+          ".hg"
+          ".bsp"
+        ];
+      };
+      colors = {
+        date = {
+          day-old = "green";
+          older = "dark_green";
+        };
+        size = {
+          none = "grey";
+          small = "grey";
+          medium = "yellow";
+          large = "dark_yellow";
+        };
+      };
+    };
+
+    test.stubs.lsd = { };
+
+    nmt.script = ''
+      assertFileExists home-files/.config/lsd/config.yaml
+      assertFileExists home-files/.config/lsd/colors.yaml
+      assertFileContent \
+        home-files/.config/lsd/config.conf \
+        ${./example-settings-expected.conf}
+      assertFileContent \
+        home-files/.config/lsd/colors.conf \
+        ${./example-colors-expected.conf}
```

Perhaps you mean
```suggestion
      assertFileContent \
        home-files/.config/lsd/config.yaml \
        ${./example-settings-expected.yaml}
      assertFileContent \
        home-files/.config/lsd/colors.yaml \
        ${./example-colors-expected.yaml}
```
?

---

My silly mistake, I copy the tests from kitty and their file extension are `*.conf`, but I forgot to update them.